### DISCUSSION TITLE
[Support] egg-ts-hepler不生成src/app中的controller和service的定义

### DISCUSSION THREAD

**Discussion by zhhbinn** [midwayjs/midway#3607] [Q&A]
### Describe the problem（描述问题）

egg-ts-hepler不生成src/app中的controller和service的定义，无法快速定位到声明和用法
![image](https://github.com/midwayjs/midway/assets/40169558/be12cee8-b44b-421f-9503-917a06440739)

### Midway Versions（Midway 版本）

midway-version@1.3.8

**Comment by czy88840616** [comment]
![image](https://github.com/midwayjs/midway/assets/418820/e31584d4-017a-4262-a2c7-36f980fae086)
对的，这里有明细的解释。

**Reply by zhhbinn** [reply]
由于是旧项目迁移至Midway，相当多的业务代码还在eggjs框架中，这块不支持生成定义导致了开发的诸多不便。方便问下不支持的原因以及后续是否能够提供支持吗？相信应该不少项目都是从eggjs迁移至Midway来，有一定的需求。感谢回答

**Reply by czy88840616** [reply]
目前 egg 已经不属于 midway 第一优先级支持的框架，不会更多的投入到 egg 的功能适配中，这也是为什么从 v3 开始就弱化并将 koa 作为默认脚手架提供。

当前 egg 项目原有代码，即使原样不动拷贝迁移过来，也是可以用的（这个由 egg 自己的逻辑保证），但是我理解是两个不同的框架，理念和技术都有差异，所以在这一层面上不会过多的去投入到 egg 了。
